dictionary.txt

```diff
@@ -93,6 +100,7 @@ CumReport
 CumulativeOutput
 customreporting
 customAdditional
+cv
 darkblotched
```

Comment:
Does darkblotched need to be defined?

Reply:
Yes, it is flagged it the latex spellcheck. This is a file of words for the latex spellcheck to ignore. Latex spellcheck is done in the github action to render the pdf. There are still a lot of things flagged in that file but I tried to get it down to as few flags as possible so that when changes are made you can look over it and see if there are any major glaring spelling/grammar errors.